test/fit_tests/deploy/os_ova_install.py

```diff
@@ -4,7 +4,11 @@
 Author(s):
 George Paulos
 
-This script installs blank OVA template.
+This script installs blank OVA templates into selected stack.
+if numvms is greater than 1, MAC address installation on OVA is done by stack number convention
+
+usage:
+    python run_tests.py -stack <stack ID> -numvms <number> -test deploy/os_ova_install.py
```

Comment:
adding usage here - do we HAVE to now specify -numvms when we use this script?
if not, can you also add example usage for the old method?
python run_tests.py -stack <stack id> -test deploy/os_ova_install.py